website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -129,3 +129,26 @@ The SideNav should always be positioned on the left side of the viewport, occupy
 On smaller viewports, the SideNav should collapse to maximize the available real estate on tablet and mobile devices. By tapping the menu icon, users can expand and access the full menu when needed.
 
 ![Responsive side-nav](/assets/components/side-nav/sidenav-position-and-responsive.png)
+
+## Collapse functionality
+
+If the `isCollapsible` property is set to true, a collapse toggle button will be exposed to the end-user allowing them to manually expand and collapse the component. 
+
+![SideNav collapse function](/assets/components/side-nav/sidenav-collapse-interaction.png)
+
+On smaller viewports the SideNav will be rendered in its collapsed state **by default** and will overlay the main page content it its expanded state.
+
+![SideNav overlay on smaller viewports](/assets/components/side-nav/sidenav-overlay-small-viewport.png)
+
+### Collapsed reflow
+
+The collapse functionality of the SideNav gives control to the end-user to unlock more horizontal space in the main page. Thus, the main page content should reflow to occupy this space if the SideNav is in it's collapsed state.
```

Comment:
HCP and TFC are trying to move towards using a centered max-width container (1064px) for the content. See [TFC's proposal](https://www.figma.com/file/Fu9P9UOQrzfrHAvP3jPX42/TFC---Patterns-and-Components?type=design&node-id=874%3A42703&mode=design&t=bjKPTSwzT6m5DXCN-1). 

I'm not sure how wide the body content container is in your example, but we should:
- Confirm with Liz if this is the same max-width of the container in HCP
- Update the example video, if needed
- Consider if we need to add details clarifying the max-width approach

Reply:
I'll connect with Liz to see if this example still makes sense or how to future proof it. Given the proposed layout changes, what do you think about making this example a bit more agnostic to the page content and including more placeholder-type visuals? This would match some of the other examples as well that @didoo mocked up in the original docs

Reply:
Ok, I've connected with Liz on this topic and made some updates/cleanup based on her recommendations.